modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -53,11 +53,24 @@
 			<argument index="0" name="s" type="float">
 			</argument>
 			<description>
-				Returns the absolute value of parameter [code]s[/code] (i.e. unsigned value, works for integer and float).
+				Returns the absolute value of parameter [code]s[/code] (i.e. unsigned value).
+				[codeblock]
+				# a is 1.0
+				a = abs(-1.0)
+				[/codeblock]
+			</description>
+		</method>
+		<method name="abs">
+			<return type="int">
+			</return>
+			<argument index="0" name="s" type="int">
+				Returns the absolute value of parameter [code]s[/code] (i.e. unsigned value).
 				[codeblock]
 				# a is 1
 				a = abs(-1)
 				[/codeblock]
+			</argument>
```

Comment:
I don't know why, but these lines, 67-71, along with lines 1115-1120, get removed with `--doctool .`. @akien-mga should take a look at this, this is weird.

Reply:
You cannot document `<argument>`. This should be under `<description>`,

Reply:
@akien-mga I have written the description in the correct place, but the behavior reported by @aaronfranke above is still there. I cannot figure out why.